src/test/java/dev/openfeature/sdk/HookSpecTest.java

```diff
@@ -503,7 +504,7 @@ public void finallyAfter(HookContext<Boolean> ctx, Map<String, Object> hints) {
         order.verify(hook2).before(captor.capture(), any());
 
         HookContext hc = captor.getValue();
-        assertEquals(hc.getCtx(), ctx);
+        assertEquals(hc.getCtx().getTargetingKey(), targetingKey);
```

Comment:
The instance of the evaluation context now changes here, but that's not something we guarantee.